.github/workflows/build.yml

```diff
@@ -46,6 +46,16 @@ jobs:
         uses: actions/checkout@v3
       - name: check manifests
         run: make check-manifests
+  diff-modules:
```

Comment:
We need to switch to Ubuntu 22.04 for all workflows. It should be done in a separate PR